source/agora/consensus/protocol/Nominator.d

```diff
@@ -490,16 +490,17 @@ extern(D):
         //    so we must collect confirm signatures regardless.
         if (envelope.statement.pledges.type_ == SCPStatementType.SCP_ST_CONFIRM)
         {
+            const value = () @trusted {
+                return envelope.statement.pledges.confirm_.ballot.value; }();
```

Comment:
What's the type of `value` ? Doesn't this make a copy of a vector ?

Reply:
I guess copying is costly.. I'll rework it.

Reply:
Wrong ping, I meant to ping you here.